tt-train/scripts/install_cmake_3_30.sh

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+sudo apt-get update && sudo apt-get upgrade -y
+sudo apt-get install build-essential libssl-dev
+CURRENT_PATH=$(pwd)
+cd /tmp
+wget https://github.com/Kitware/CMake/releases/download/v3.30.0/cmake-3.30.0.tar.gz
+tar -zxvf cmake-3.30.0.tar.gz
+cd cmake-3.30.0
+./bootstrap
+make -j$(nproc)
```

Comment:
Why not use pre-built binaries?

Also: building from source with defaults lacked TLS support (or at least old versions did), which caused confusing failures when fetching content.

Reply:
wait what? why? why does this script exist if cmake 3.20 seems to work in our CI?

Reply:
we could not get prebuilt version for the 3_30. But if 3_20 works we can ignore it.